src/components/Integrations/Shopify/Shopify.test.js

```diff
@@ -5,6 +5,11 @@ import DopplerIntlProvider from '../../../i18n/DopplerIntlProvider.double-with-i
 import { AppServicesProvider } from '../../../services/pure-di';
 import Shopify from './Shopify';
 import { SubscriberListState } from '../../../services/shopify-client';
+import {
+  experimentalFeatures,
+  ExperimentalFeatures,
```

Comment:
What are these both things with pretty similar names?

Reply:
A mistype or perhaps autocode fix, I will remove this right away thanks!